web/docs/introduction/introduction.md

```diff
@@ -193,7 +192,7 @@ Wasp is addressing the same core problems that typical web app frameworks are ad
 You don't need to know what a DSL is to use Wasp, but if you are curious, you can read more about it below.
 :::
 
-Wasp does not match typical expectations of a web app framework: it is not a set of libraries, it is instead a programming language that understands your code and can do a lot of things for you.
+Wasp does not match typical expectations of a web app framework: it is not a set of libraries, it is instead a simple programming language that understands your code and can do a lot of things for you.
```

Comment:
Do we maybe want to delang this?

Reply:
I think so, but let's do it later properly, I did a tiny change here :D.

I actually think it is ok to leave this here for now, until we potentially introduce TS SDK, then we can rephrase it (although I would love mention of DSL, but we can talk a bit less about Wasp being a prog lang).